example/full-config.php

```diff
@@ -61,7 +61,6 @@
             'orm_default' => [
                 'driver_class' => Driver::class,
 //                'wrapper_class' => null,
-                'pdo' => null,
                 'configuration' => 'orm_default', // Actually defaults to the connection config key, not hard-coded
                 'params' => ['serverVersion' => 'mariadb-11.0'],
```

Comment:
Can we add a full example of `primary` and `replica` usage here too?

Reply:
I've added a separate example for primary-replica connection.